docs/data-sources/cbr_vaults.md

```diff
@@ -0,0 +1,105 @@
+---
+subcategory: "Cloud Backup and Recovery (CBR)"
+---
+
+# huaweicloud_cbr_vault
+
+Use this data source to get available CBR vaults within Huaweicloud.
+
+## Example Usage
+
+### Obtain the vault list by vault name
+
+```hcl
+variable "vault_name" {}
+
+data "huaweicloud_cbr_vaults" "test" {
+  name = var.vault_name
```

Comment:
how about this:
`
data "huaweicloud_cbr_vaults" "test" {
  name = "your_cbr_vault_name"
}
`

Reply:
To be debated.